### PR TITLE
Add EventsQuery

### PR DESCRIPTION
**Pull request by kulla**
@inyono I want to go through this PR tomorrow. There might be some small changes I will do. However it's good enough to get some first feedback :smile:

### CODE REVIEW

File: src/event/model/event.rs [hunk 1]
```diff
@@ -251,3 +253,84 @@ impl EventPayload {
...
+        after: i32,
+        max_events: i32,
+        pool: &MySqlPool,
+    ) -> Result<Events, sqlx::Error> {
+        let event_ids = Events::fetch_event_ids(after, max_events, pool).await?;
+
+        let mut event_tasks = FuturesOrdered::new();
+        for event_id in event_ids {
+            event_tasks.push(Events::fetch_event(event_id, pool));
+        }
```
Comment (inyono) [inline src/event/model/event.rs:273]: That looks super inefficient. Doesn't this trigger `max_events` sql queries in sequence?

Comment (kulla) [reply src/event/model/event.rs:273]: Yep :see_no_evil: -> Let's see whether there is a bette impl

**Comment by inyono** [comment]
Any news on this PR? Do you need help with the query or something?

**Comment by kulla** [comment]
> Any news on this PR? Do you need help with the query or something?

No, it's my next task to go further...

**Comment by kulla** [comment]
@inyono suggestions implemented. ready for review

**Review by inyono — APPROVED** [review]
LGTM

**Pull request merged**